lms/djangoapps/ccx/tests/test_utils.py

```diff
@@ -3,17 +3,28 @@
 """
 from nose.plugins.attrib import attr
 
+from django.conf import settings
 from lms.djangoapps.ccx.tests.factories import CcxFactory
 from student.roles import CourseCcxCoachRole
 from student.tests.factories import (
     AdminFactory,
+    CourseEnrollmentFactory,
+    UserFactory
 )
 from xmodule.modulestore.tests.django_utils import (
     ModuleStoreTestCase,
+    SharedModuleStoreTestCase,
     TEST_DATA_SPLIT_MODULESTORE)
 from xmodule.modulestore.tests.factories import CourseFactory
+from xmodule.modulestore.django import modulestore
+
+from student.roles import CourseCcxCoachRole, CourseStaffRole
+
+from instructor.access import list_with_level, allow_access
 
 from ccx_keys.locator import CCXLocator
+from lms.djangoapps.ccx.overrides import override_field_for_ccx
+from ..utils import add_master_course_staff_to_ccx, ccx_course
```

Comment:
can please you use the full path for this? Even if it is correct I would avoid using this kind of imports unless the module is in the same directory of the file that imports it